app/cafe.py

```diff
@@ -0,0 +1,21 @@
+from datetime import date
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("The guest without vaccine")
+        elif visitor["vaccine"]["expiration_date"] < date.today():
+            raise OutdatedVaccineError("The vaccine is outdated")
```

Comment:
it's better to use `if` here instead of `elif` because logically few errors can happen at the same time and interpreter should access all blocks of code